backend/open_api_spec.yml

```diff
@@ -0,0 +1,829 @@
+                schema:
+                  $ref: "#/components/schemas/collection_uuid"
+        "401":
+          $ref: "#/components/responses/401"
+
+  /curation/collections/{collection_id}:
+    delete:
+      description: Delete a private Collection. Delete an existing revision for a public Collection.
+      security:
+        - cxguserCookie: []
+      parameters:
+        - $ref: "#/components/parameters/path_collection_uuid"
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              type: string
+        "401":
+          $ref: "#/components/responses/401"
+        "404":
+          $ref: "#/components/responses/404"
+    get:
+      description: Fetch Collection metadata and associated Datasets, public and private.
+      security:
+        - cxgUserCookieLenient: []
+      parameters:
+        - $ref: "#/components/parameters/path_collection_uuid"
+        - $ref: "#/components/parameters/query_visibility"
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              $ref: "#/components/schemas/collection"
+        "401":
+          $ref: "#/components/responses/401"
+        "404":
+          $ref: "#/components/responses/404"
+    put:
+      description: Start a revision. Update any included Collection metadata fields.
+      security:
+        - cxguserCookie: []
+      parameters:
+        - $ref: "#/components/parameters/path_collection_uuid"
+      requestBody:
+        content:
+          application/json:
+            $ref: "#/components/schemas/collection_metadata"
+      responses:
+        "204":
+          $ref: "#/components/responses/204"
+        "401":
+          $ref: "#/components/responses/401"
+        "404":
+          $ref: "#/components/responses/404"
+
+  /curation/collections/{collection_id}/datasets:
+    delete:
+      description: >
+        Delete a private Dataset. For a public Dataset, start a revision if
+        one does not exist, then delete the Dataset from the revision.
+      security:
+        - cxguserCookie: []
+      parameters:
+        - $ref: "#/components/parameters/path_collection_uuid"
+        - $ref: "#/components/parameters/optional_query_curator_tag"
+        - $ref: "#/components/parameters/optional_query_dataset_uuid"
+        - $ref: "#/components/parameters/query_visibility"
+      responses:
+        "204":
+          $ref: "#/components/responses/204"
+        "401":
+          $ref: "#/components/responses/401"
+        "404":
+          $ref: "#/components/responses/404"
+    get:
+      description: Fetch a Dataset.
+      security:
+        - cxguserCookie: []
+      parameters:
+        - $ref: "#/components/parameters/path_collection_uuid"
+        - $ref: "#/components/parameters/optional_query_curator_tag"
+        - $ref: "#/components/parameters/optional_query_dataset_uuid"
+        - $ref: "#/components/parameters/query_visibility"
```

Comment:
So the curator can fetch a private (revision) version of a public dataset.

Reply:
Aren't these all returned when they do `GET /collections/{collection_id}?visibility=private`? I think this endpoint is unnecessary.

Reply:
I agree it isn't necessary, just adds the ability to filter before launching the request. May be nice if a collection has hundreds of datasets, lower response time.

Reply:
We do not have collections with _hundreds of datasets_. The largest has 41, but most are much more limited.

Reply:
not needed